app/services/flags/destroy_flag.rb

```diff
@@ -16,7 +15,7 @@ class Flags::DestroyFlag
   private
 
   def fetch_flag(id:)
-    Flag.find(id)
+    Flag.find_by(id: id)
```

Comment:
We don’t want to use `find` here, as it will raise an error if the model is not found. This will break the execution flow of the service.